data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,450 @@
+			label wait
+			`	You are having a hard time hearing anything with the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm.`
+			`	They turn to you, surprisingly perfectly calm in this situation. "Lucky for you, this is our ship. But not any ship! This is a mighty relic of the past, from the times our Empire used to control the stars: a Ladybug!`
+			choice
+				`	(Say nothing.)`
+					goto explain
+				`	"You could have told me before."`
+			`	"Yes, but where is the fun in that? We also wanted to see how you would react.`
+			label explain
+			`	"We have a very interesting proposition for you: should you agree to bring supplies to our brethren, the Unfettered Hai, we will sell this ship to you for the modest price of 6.5 million credits. This is a very useful ship, and you should be able to repay this quite easily by running more of the same kind of jobs. These supplies are needed on short notice, on <destination>, and we lack the time and personnel to bring them."`
+			`	"Of course, the ship has some... drawbacks, compared to its golden time, and you should not expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I have no interest in this deal."`
+			`	"A shame. We will probably not meet again," they say, before going on the ship and putting it back in the ocean.`
+				goto trainback
+			label take
+			action
+				give ship "Ancient Ladybug" "Good Omen"
+				fine 6500000
+				log "Accepted a somewhat shady deal for an old Hai ship, a Ladybug, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go trough official means for this, we will have to present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you notice the fine has already been issued before you could say anything. "I would advise against repainting the ship, you will understand why later. Also make sure you go to the spaceport to get the cargo loaded onto your ship!"`
+				accept
+			label dumb
+			`	You get your weapon out and proceed to shoot on the ship.`
+			branch lucky?
+				random < 2
+			branch lucky??
+				random < 40
+			`	You shoot the ship, but the small weapon has no chance of doing anything to the heavy armor, even in this state. The ship's speakers then turn up. "MY TURN!"`
```

Comment:
Turn is used twice, not sure how it could be better, though.

Reply:
```suggestion
			`	You shoot the ship, but the small weapon has no chance of doing anything to the heavy armor, even in this state. The ship's speakers then activate, almost breaking your ears with their volume. "MY TURN!"`
```
?

Reply:
Yes!